README.md

```
@@ -1,63 +1,50 @@
-<a href="https://f-droid.org/packages/zed.rainxch.githubstore">
-  <img src="https://img.shields.io/f-droid/v/zed.rainxch.githubstore?color=a1168e&include_prereleases&logo=FDROID&style=for-the-badge&labelColor=700f63"/>
-</a>
-
-<br/>
-<br/>
-
+  <img alt="Kotlin" src="https://ziadoua.github.io/m3-Markdown-Badges/badges/Kotlin/kotlin1.svg" />
+  <img src="https://api.github-store.org/v1/badge/static/11/2?label=Apache--2.0&icon=palette" alt="Apache-2.0"/>
+  <br/>
+  <br/>
+  <img src="https://ziadoua.github.io/m3-Markdown-Badges/badges/Android/android1.svg" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)